app/models/concerns/price_validator.rb

```diff
@@ -0,0 +1,8 @@
+class PriceValidator < ActiveModel::Validator
```

Comment:
why in concern folder? where it should be

Reply:
it should be in validators folder as it makes more sense.